### DISCUSSION TITLE
Add code completion for groups for `is_in_group()` etc

### DISCUSSION THREAD

**Discussion by me2beats** [godotengine/godot-proposals#4341] [Editor]
It would be useful to have code completion for methods like `is_in_group()`,  `get_nodes_in_group()` etc

**Comment by Calinou** [comment]
This would require groups to be registered globally in the project somewhere, which is not the case yet: https://github.com/godotengine/godot-proposals/issues/3789

**Reply by theraot** [reply]
They are on the SceneTree, aren't they? The editor could auto complete those on the currently edited scene. Similar to what happens with NodePath literals, that the editor only autocompletes if the currently edited scene has the script where I'm typing.

Edit: a global registry would probably better.
